bucket/dwarf-fortress.json

```diff
@@ -15,6 +15,10 @@
         "url": "http://www.bay12games.com/dwarves",
         "re": "DWARF FORTRESS CLASSIC ([\\d\\.]+)"
     },
+    "persist": [
+        "data",
+        "data"
+    ],
```

Comment:
Why is "data" written twice here?

Reply:
I was going off of this in the documentation

```
 {
    "persist": [
        "keeps_its_name",
        ["original_name", "new_name_inside_the_data_dir"]
    ]
}
```

Reply:
If you want the rename behavior to occur, it needs to be an array *within* another array. Also, since the names are identical, I don't think you need it here :slightly_smiling_face:

Reply:
I applied the change directly.